kernel/kernel_config-4.9.x

```diff
@@ -243,16 +243,13 @@ CONFIG_OPROFILE=y
 # CONFIG_OPROFILE_EVENT_MULTIPLEX is not set
 CONFIG_HAVE_OPROFILE=y
 CONFIG_OPROFILE_NMI_TIMER=y
-CONFIG_KPROBES=y
+# CONFIG_KPROBES is not set
```

Comment:
Is KPROBES problematic? Does it stop `perf` from working? We definitely would like to enable perf debugging on kernels